### DISCUSSION TITLE
Built with Gatsby 🖥

### DISCUSSION THREAD

**Discussion by davemullenjnr** [gatsbyjs/gatsby#29459] [Community]
I was looking for a thread of websites that have been built with Gatsby, so thought I'd start one.

A nice format to stick to would be:
- Title
- Link to website
- Link to repo (optional)
- Talk about it
- Stack (notable contributions)

**Comment by davemullenjnr** [comment]
**HoodieHut**
[hoodiehut.co.uk](https://www.hoodiehut.co.uk)
—
This isn't my first website built with Gatsby but it's been (by far) the most comprehensive. Really got to grips with **gatsby-image** and **GraphQL** and thoroughly enjoyed the developer experience.
—
CMS: [NetlifyCMS](https://www.netlifycms.org)
Hosting: [Netlify](https://www.netlify.com)
CSS: [Styled Components](https://styled-components.com)
Carousel: [Embla](https://davidcetinkaya.github.io/embla-carousel/)

**Reply by rodneylab** [reply]
Looks great, clean design and has the speed you expect from a Gatsby site 💯

**Comment by herecydev** [comment]
Pretty close to what you're asking for is: https://www.gatsbyjs.com/showcase/

**Reply by davemullenjnr** [reply]
Hi @herecydev — Yeah, I know about Showcase but as of writing they say:

> Note: Showcase and starter contributions are currently paused while the Gatsby team develops a new workflow for sharing your Gatsby sites with the community.

I came here from [Spectrum.chat](https://spectrum.chat/gatsby-js?tab=posts) where there was a feed like this and people would share their Gatsby sites :)
